contrib/cirrus/runner.sh

```diff
@@ -19,6 +19,7 @@ _run_noarg() {
 _run_build() {
     # Assume we're on a fast VM, compile everything needed by the
     # rest of CI since subsequent tasks may have limited resources.
+	make vendor
```

Comment:
Do we need these? Shouldn't Cargo automatically fetch deps?

Reply:
they were needed or the sources were not present.